### DISCUSSION TITLE
When using Email provider, is it possible to specify  certain emails to access?

### DISCUSSION THREAD

**Discussion by chloesun** [nextauthjs/next-auth#1727] [Help] [answered]
I'm building an internal site, and it should only be accessible to certain users. Is it possible to specify certain emails that can be signed in through Email provider, so the admin can manage who can access the site?

I'm not sure if this is possible for the passwordless tech, or I need to use username/password approach.

Thanks for your input!

**Comment by StefanSelfTaught** [comment, marked as answer]
You can try to use the signIn callback like this:
```js
async signIn(user, account) {
  if (account.type === 'email' && user.email = 'user@email.com') {
    return true
  }
  return false
}
